src/components/FormField/Input/TextField.tsx

```diff
@@ -0,0 +1,63 @@
+import { InputHTMLAttributes, forwardRef, useId, useRef } from "react";
+import { Icon } from "../..";
+import { IconButton, InputElement, InputWrapper, WrapperProps } from "./InputWrapper";
+import { mergeRefs } from "@/utils/mergeRefs";
+import { mockInputChangeEvent } from "./mockInputChangeEvent";
+interface TextInputProps
+  extends Omit<WrapperProps, "id" | "children">,
+    Omit<InputHTMLAttributes<HTMLInputElement>, "children" | "type" | "value"> {
+  type?: "text" | "email" | "tel" | "url";
+  loading?: boolean;
+  value?: string;
+  clear?: boolean;
+}
+
+export const TextField = forwardRef<HTMLInputElement, TextInputProps>(
+  (
+    { clear, type, disabled, label, error, id, loading, value = "", onChange, ...props },
+    ref
+  ) => {
+    const inputRef = useRef<HTMLInputElement>(null);
+    const defaultId = useId();
+    const clearInput = () => {
+      mockInputChangeEvent(inputRef.current, onChange);
```

Comment:
let's simply pass an empty value

Reply:
if we want to be consistent, we should let the client control the input and therefore we could pass the value.

maybe we could pass the event to the onChange function as a second argument if needed. wdyt @vineethasok @fneves?